File: Src2/Icm.ContextConsole/NamesSynonyms/SynonymAttribute.vb
''' <summary>
''' This attribute provides synonyms for the given item.
''' </summary>
''' <remarks></remarks>
Public Class SynonymAttribute
    Inherits Attribute

    Public Synonyms() As String

    Public Sub New(ParamArray synonyms As String())
        Me.Synonyms = synonyms
    End Sub

End Class

File: Src/Icm.ContextConsole/NamesSynonyms/SynonymAttribute.cs
using System;

/// <summary>
/// This attribute provides synonyms for the given item.
/// </summary>
/// <remarks></remarks>
public class SynonymAttribute : Attribute
{


	public string[] Synonyms;
	public SynonymAttribute(params string[] synonyms)
	{
		this.Synonyms = synonyms;
	}

}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
